pkgs/development/python-modules/imagecodecs/default.nix

```diff
@@ -0,0 +1,85 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cython,
+  numpy,
+  setuptools,
+  pkgs,
+  jxrlib,
+  lcms2,
+  lerc,
+  libdeflate,
+  libpng,
+  libtiff,
+  libwebp,
+  openjpeg,
+  xz,
+  zlib,
+  zstd,
+}:
+
+let
+  version = "2025.3.30";
+in
+buildPythonPackage {
+  pname = "imagecodecs";
+  inherit version;
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "cgohlke";
+    repo = "imagecodecs";
+    tag = "v${version}";
+    hash = "sha256-KtrQNABQOr3mNiWOfaZBcFceSCixPGV8Hte2uPKn1+k=";
+  };
+
+  build-system = [
+    cython
+    numpy
+    setuptools
+  ];
+
+  nativeBuildInputs = [
+    pkgs.lz4.dev # lz4 was hidden by python3Packages.lz4
+    lcms2.dev
+    openjpeg.dev
+  ];
+
+  buildInputs = [
+    pkgs.lz4
+    jxrlib
+    lcms2
+    lerc
+    libdeflate
+    libpng
+    libtiff
+    libwebp
+    openjpeg
+    xz # liblzma
+    zlib
+    zstd
+  ];
+
+  dependencies = [
+    numpy
+  ];
+
+  prePatch = ''
+    substituteInPlace setup.py \
+      --replace-fail "/usr/include/openjpeg" "${openjpeg.dev}/include/openjpeg" \
+      --replace-fail "/usr/include/jxrlib" "${jxrlib}/include/jxrlib"
+  '';
+
+  pythonImportsCheck = [
+    "imagecodecs"
+  ];
+
```

Comment:
Can you please check if there are tests?
If not, perhaps add a note on why tests are not run.